### PR TITLE
Fix: Floating Buttons - fixing CSS for floating buttons preview

### PR DESCRIPTION
**Pull request by mserino**
## PR Checklist

- [ ] The commit message follows our guidelines:  https://github.com/elementor/elementor/blob/master/.github/CONTRIBUTING.md

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] Other... Please describe:

## Summary

This PR can be summarized in the following changelog entry:

*

## Description
An explanation of what is done in this PR

*

## Test instructions
This PR can be tested by following these steps:

*

## Quality assurance

- [ ] I have tested this code to the best of my abilities
- [ ] I have added unittests to verify the code works as intended
- [ ] Docs have been added / updated (for bug fixes / features)

Fixes #

### CODE REVIEW

File: assets/dev/scss/editor/_templates-modal.scss [hunk 1]
```diff
@@ -828,6 +828,11 @@ label.elementor-template-library-order-label {
 	margin-block-start: 10px;
 }
 
+.elementor-template-library-template-remote:not(.elementor-template-library-template-page):not(.elementor-template-library-template-lp).elementor-template-library-template-floating_button {
+	margin-top: 0 !important; // to override computed inline styles
+	width: calc(25% - 30px);
+}
```
Comment (louiswol94) [inline assets/dev/scss/editor/_templates-modal.scss:834]: Can you add this CSS in the floating buttons module?

File: modules/floating-buttons/assets/scss/widgets/contact-buttons-var-4.scss [hunk 2]
```diff
@@ -78,16 +78,21 @@
...
+			}
 
 			svg {
 				fill: var(--e-contact-buttons-contact-button-icon);
 			}
 
 			&:hover,
 			&:focus {
-
+
```
Comment (louiswol94) [inline modules/floating-buttons/assets/scss/widgets/contact-buttons-var-4.scss:92]:
```suggestion

```
Undo

**Review by louiswol94 — CHANGES_REQUESTED** [review]

File: assets/dev/scss/editor/_templates-modal.scss (hunk 1 above)
Comment (mserino) [reply assets/dev/scss/editor/_templates-modal.scss]: done

File: modules/floating-buttons/assets/scss/widgets/contact-buttons-var-4.scss (hunk 2 above)
Comment (mserino) [reply modules/floating-buttons/assets/scss/widgets/contact-buttons-var-4.scss]: done

**Review by louiswol94 — APPROVED** [review]

**Pull request merged**